github.yaml

```diff
@@ -58,6 +58,52 @@ systems:
             - name: resource_path
               description: This field is used as the path portion of the API call url
 
+      createPR:
+        target:
+          system: github
+          function: api
+        parameters:
+          URL: 'https://api.github.com/repos/{{ .ctx.git_repo }}/pulls'
+          method: POST
+          content: $ctx.PR_content
+        export:
+          PR: '{{ .data.json }}'
+
+        description: >
+          This function will create a pull request with given infomation
+
+        meta:
+          inputs:
+            - name: git_repo
+              description: The repo that the new PR is for, e.g. :code:`myorg/myrepo`
```

Comment:
Why this instead of splitting up org and repo name?

Reply:
The function itself doesn't really need the org and repo name separately.  It doesn't add any benefit to make them two separate variable.  However, users can store them as separate variables if they desire, only need to combine them when they invoke the function.